python/tools/manage_project_schemas/README.rst

```diff
@@ -49,3 +49,51 @@ Dependencies
 ------------
 
 * ftrack_api
+
```

Comment:
@nebukadhezer could you convert the rst to md for github ? 
is now the default format we use for these files.
Thanks.